server/subscription.go

```diff
@@ -68,6 +52,32 @@ func (s *Server) registerSubscriptionRoutes(g *echo.Group) {
 	})
 }
 
+// loadLicense will load current subscription by license.
+// Return subscription with free plan if no license found.
+func (server *Server) loadSubscription() *enterprise.Subscription {
+	subscription := &enterpriseAPI.Subscription{
+		Plan: api.TEAM,
```

Comment:
should this be free?

Reply:
Just to align with old logic. I think we can set it to TEAM in dev mode. WDYT

Reply:
LGTM